tests/server/amplitude.js

```diff
@@ -46,6 +47,14 @@ registerSuite('amplitude', {
       assert.lengthOf(amplitude, 3);
     },
 
+    'disable writing amplitude events if amplitude.diabled is true': () => {
```

Comment:
Your comment seems right, but in the test case you are not actually sending an amplitude event. See the `flow.signin_from.bar` test below for an basic example that sends an amplitude event and then tries to see what was logged.

In the disabled case, nothing should have been logged.

Reply:
@vbudhram thanks for correcting me.